packages/registry-contracts/script/test/deploy-protocol-controller.sh

```
@@ -0,0 +1,64 @@
+  echo "Deploy the verifier contracts first with deploy-root-verifier.sh and export ROOT_VERIFIER_ADDRESS"
+  exit 1
+fi
+
+# Deploy the ProtocolController contract
+echo "Deploying ProtocolController..."
+forge script script/DeployProtocolController.s.sol:DeployProtocolControllerScript \
+  --rpc-url $RPC_URL \
+  --private-key $DEPLOYER_PRIVATE_KEY \
+  --broadcast
+
+export PROTOCOL_CONTROLLER_ADDRESS=$(cat broadcast/DeployProtocolController.s.sol/$CHAIN_ID/run-latest.json | jq -r '.transactions[] | select(.transactionType=="CREATE") | .contractAddress')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)